azurerm/resource_arm_subnet.go

```diff
@@ -257,6 +267,12 @@ func resourceArmSubnetRead(d *schema.ResourceData, meta interface{}) error {
 	if props := resp.SubnetPropertiesFormat; props != nil {
 		d.Set("address_prefix", props.AddressPrefix)
 
+		var privateLinkServiceNetworkPolicies *string
+		if props.PrivateLinkServiceNetworkPolicies != nil {
+			privateLinkServiceNetworkPolicies = props.PrivateLinkServiceNetworkPolicies
```

Comment:
These lines of code don't actually do anything as privateLinkServiceNetworkPolicies defaults to nil...

Reply:
This has already been changed...